clients/tabby-agent/src/chat/smartApply.ts

```diff
@@ -130,13 +125,13 @@ export class SmartApplyFeature implements Feature {
       logger.error("Error applying smart edit:", error);
       return false;
     } finally {
-      logger.info("Resetting mutex abort controller");
+      logger.debug("Resetting mutex abort controller");
       this.mutexAbortController = undefined;
     }
   }
 }
 
-export async function provideSmartApplyLineRange(
+async function provideSmartApplyLineRange(
```

Comment:
There is no need to export a function that is only used in the same file. This is similar to preferring `private` over `public`.